src/api/access-types.js

```diff
@@ -29,19 +30,19 @@ export default {
 
     return doRequest(url, params);
   },
-  deleteAccessType: (okapi, accessType) => {
+  deleteAccessType: (okapi, accessType, credentialId) => {
     const method = 'DELETE';
-    const url = `${okapi.url}${API_URL}/${accessType.id}`;
+    const url = `${okapi.url}${credentialId ? CREDENTIALS_API_URL(credentialId) : API_URL}/${accessType.id}`;
     const params = {
       method,
       headers: getHeaders(method, okapi, url),
     };
 
     return doRequest(url, params);
   },
-  updateAccessType: (okapi, accessType) => {
+  updateAccessType: (okapi, accessType, credentialId) => {
     const method = 'PUT';
-    const url = `${okapi.url}${API_URL}/${accessType.id}`;
+    const url = `${okapi.url}${credentialId ? CREDENTIALS_API_URL(credentialId) : API_URL}/${accessType.id}`;
```

Comment:
I think `credentialId ? CREDENTIALS_API_URL(credentialId) : API_URL` could be a single variable, you use it 4 times
for example: 
``getApiUrl = credentialId => credentialId ? `/eholdings/kb-credentials/${credentialId}/access-types` : API_URL``